.github/pull_request_template.md

```diff
@@ -1,14 +1,14 @@
-*By submitting this pull request you agree that all contributions to this project are made under the MIT license.*
-
-## Issues
-
-<!-- Link the issues this change resolves, or describe them -->
+<!-- By submitting this pull request you agree that all contributions to this project are made under the MIT license. -->
 
 ## Summary
 
 <!-- Describe how these changes resolve the aforementioned issues -->
 
+## Issues
+
+<!-- Link the issues this change resolves, or describe them -->
+
```

Comment:
My intention with the order here was to first have people describe the problem (an issue link being sufficient) and then summarize how these changes fix it. We can change the order but the comments describing the purpose of each section need to be updated to reflect that.